.eslintrc.json

```diff
@@ -1,17 +1,35 @@
 {
     "env": {
-        "browser": true,
-        "commonjs": true,
-        "es6": true
+        "es6": true,
+        "node": true,
+        "mocha": true
     },
     "extends": "eslint:recommended",
-    "globals": {
-        "Atomics": "readonly",
-        "SharedArrayBuffer": "readonly"
+    "rules": {
+        "indent": [
+            "error",
+            2,
+            {
+                "SwitchCase": 1
+            }
+        ],
+        "no-console": "off",
+        "no-var": "error",
+        "prefer-const": "error",
```

Comment:
I guess this line results in an error for each variable which is not 'const'.
Unless there is really proof of an advantage in speed or cause of errors, I consider this as a semantics discussion.
Everyone has its coding style, this is mine, sorry.

I've seen some useful changes below (thanks to take the effort!!) and will implement them, but I'll reject this change request.

Reply:
How do you want to proceed? I can also publish my version to npm. Since I need that changes for my own ioBroker adapter.